src/Glpi/Kernel/Kernel.php

```diff
@@ -153,6 +154,14 @@ protected function configureRoutes(RoutingConfigurator $routes): void
         if (\is_file($path = $this->getProjectDir() . '/routes/' . $this->environment . '.php')) {
             (require $path)($routes->withPath($path), $this);
         }
+
+        // Plugin-specific routes
+        $routes->add(PluginsRouterListener::ROUTE_NAME, '/{plugins_or_marketplace}/{plugin_name}{path_rest}')
+            ->requirements([
+                'plugins_or_marketplace' => '^(plugins|marketplace)$',
```

Comment:
As suggested by @trasher in our internal chat, we should only support only `/Plugin` here no ?
We don't need them the internal folder be to "known" in the url.

Unless this is also used for legacy backward compatibility, but from my understanding this would only be for proper controllers created in plugins (the plugin legacy front files are handled by the same legacy stuff that for handle the ones for the core right ?).

Reply:
I'm not fond of uppercase letters in URLs (they often are confusing).

However, I'll indeed refactor this and only use `/plugin/{plugin_name}` as matching URL, to standardize everything. This stays compatible with legacy, and even legacy URLs could just redirect to new routes by themselves if necessary.